app/services/backup_service.rb

```diff
@@ -143,6 +143,7 @@ def download_to_tar(tar, attachment, filename)
       end
     end
   rescue Errno::ENOENT
+  rescue Seahorse::Client::NetworkingError
```

Comment:
Why are we getting a Seahorse error instead of a `::Fog::Errors::Error`, which paperclip rescues by default? This feels like a `fog` bug

Reply:
Paperclip doesn't use fog for S3.